attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -66,6 +97,7 @@ impl Attester for TdxAttester {
             .map_err(|e| anyhow!("Serialize TDX evidence failed: {:?}", e))
     }
 
+    #[cfg(feature = "tdx-getquote-ioctl")]
```

Comment:
So TSM does not support extending the RTMRs? What happens if someone calls this function without the ioctl feature enabled?

Reply:
currently no, I believe the fall back is the trait default?